pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -61,6 +61,16 @@ in rec {
       };
     } ./flit-build-hook.sh) {};
 
+  fixPythonDepsHook = callPackage ({ requirements-parser }:
```

Comment:
I don't think we care, but this will definitively not work in Python 2.

Should I limit Python 2 usage with `disabledIf (!isPy3k)` like the venvShellHook?

Reply:
yes, please. Or `pythonOlder` if it requires some newer version above 3.4.